androidApp/src/main/java/dev/johnoreilly/confetti/ui/Bookmark.kt

```diff
@@ -0,0 +1,36 @@
+package dev.johnoreilly.confetti.ui
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.outlined.Bookmark
+import androidx.compose.material.icons.outlined.BookmarkAdd
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun Bookmark(
+    isBookmarked: Boolean,
+    onBookmarkClick: () -> Unit,
```

Comment:
Nit: `onBookmarkChange: (Boolean) -> Unit` where `Boolean` is the flip value, similar to how a Toggle button does in Compose.